apps/console/src/features/applications/pages/application-edit.tsx

```diff
@@ -441,7 +463,11 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
      * Handles the back button click event.
      */
     const handleBackButtonClick = (): void => {
-        history.push(AppConstants.getPaths().get("APPLICATIONS"));
+        if(!isConnectedAppsRedirect){
```

Comment:
```suggestion
        if (!isConnectedAppsRedirect) {
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3494/commits/4fada76b61b748e97d1be72a91e31eea5268bccf